### DISCUSSION TITLE
Randomize LoRA and LoRA values.

### DISCUSSION THREAD

**Discussion by spiritparticle** [innightwolfsleep/stable-diffusion-webui-randomize#3] [Ideas]
I think this would really be a great feature where you could list (or click) LoRAs to use and then set a max, min and steps for the lora amount. Random LoRAs would be an insanely great rolling the dice feature to add to the randomisation.

**Comment by innightwolfsleep** [comment]
There is trick for you)
Use [dynamic promt extension](https://github.com/adieyal/sd-dynamic-prompts)
Istall and write in prompt something like this:
`{<lora:gachaSplashLORA_gachaSplash31:1>|<lora:animeTarotCardArtStyleLora_v20Offset:1>}`
and you will get random choise between 2 loras.

(randomize is rather work with not "in-prompt" paremeters)

**Reply by emersound** [reply]
how about setting random weights for words, or for wildcards?
like `(person a:person b:min,max)` should I assume that it will generate a number between the min and max and put it in the prompt when generated?
